tests/SampleApps/DotNetCore/TodoAppFromMac/Readme.md

```diff
@@ -0,0 +1,3 @@
+This is a regularly(not self-contained) published ASP.NET Core 3.0 app on a Mac machine.
```

Comment:
This is a published app from a MacOS that I am checking in for test verification

Reply:
I think we should just build the app using `-r mac` instead of adding binaries to our repo

Reply:
I am pasting the following content from another PR for reference...basically we cannot test the scenario that I am testing here when using `-r osx-x64` as it throws:

**Comment from my old PR:**
I tried the following scenario now - publish the app for Mac and do a `dotnet foo.dll` on linux...
and this failed to execute...so we need not worry about this scenario as it would never have worked.

FYI:
**Publish for Mac but run on Linux**
![image](https://user-images.githubusercontent.com/6559784/57771910-f9eb6080-76c8-11e9-992f-6d9bd2d9dadb.png)

**Publish for Linux and run on Linux**
![image](https://user-images.githubusercontent.com/6559784/57771972-256e4b00-76c9-11e9-83c3-3273a95db973.png)